en/components/radial-gauge.md

```diff
@@ -47,58 +57,13 @@ The Radial Gauge comes with styling flexibility through the various overrides co
 | <img class="responsive-img" src="../images/radial_gauge_do.png" srcset="../images/radial_gauge_do@2x.png 2x" /> | <img class="responsive-img" src="../images/radial_gauge_dont.png" srcset="../images/radial_gauge_dont@2x.png 2x" /> |
 
-## Code Generation
-
-This section describes some important overrides and how they affect code generation.
-
-> [!WARNING]
-> Triggering `Detach from Symbol` on an instance of the Radial Gauge in your design is very likely to result in loss of code generation capability for the Radial Gauge.
-
-### Minimum Value
-
-This override controls the minimum value displayed by the Radial Gauge. It supports [data binding](../codegen/data-binding.md) as well as direct setting of a number. If a binding is used the bound property must result in a number at runtime. If no binding is used the value must be a number. If the value provided is not in the correct format the Radial Gauge will be generated without a minimum value set and the control default will be used.
-
-Binding:
-
-<img class="responsive-img" src="../images/linear_gauge_bind_min.png"/>
-
-No binding:
-
-<img class="responsive-img" src="../images/linear_gauge_nobind_min.png"/>
-
-### Maximum Value
-
-This override controls the maximum value displayed by the Radial Gauge. It supports [data binding](../codegen/data-binding.md) as well as direct setting of a number. If a binding is used the bound property must result in a number at runtime. If no binding is used the value must be a number. If the value provided is not in the correct format the Radial Gauge will be generated without a maximum value set and the control default will be used.
-
-Binding:
-
-<img class="responsive-img" src="../images/linear_gauge_bind_max.png"/>
-
-No binding:
-
-<img class="responsive-img" src="../images/linear_gauge_nobind_max.png"/>
-
-### Ranges Amount
-
-This override controls how many ranges are displayed within the Radial Gauge. During code generation it will create `<igx-radial-gauge-range>` elements which are what represents a single range. This range element has `startValue`, `endValue` and `brush` properties which are used to define where the range starts and ends and what color it is. These are all automatically generated for you such that regardless of what the Minimum Value and Maximum Value overrides are set to, you will always get the same appearance as what is displayed in the Sketch.
-
-You may not want the ranges to use the auto-generated start and end values, however, in that case you will need to manually update the ranges in the generated HTML.
-
-### ValueDataProperty
-
-This override controls the needle value displayed by the Radial Gauge. It only supports [data binding](../codegen/data-binding.md) so provided values must be in the correct format.
-
-<img class="responsive-img" src="../images/linear_gauge_valueprop.png"/>
-
-The bound property **must** result in a number at runtime.
-
 ## Additional Resources
 
 Related topics:
```

Comment:
Replace category chart with references to column and line chart and remove data-binding from the related topics.